examples/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumb/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  BreadcrumbContext,
+  BreadcrumbProps,
+} from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/types';
+import { breadcrumbContext } from './breadcrumb.context';
+import { baseBreadcrumbTheme } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/base-theme';
+import { useMemo } from 'react';
+import { pickOuterProps } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/helpers/pick-outer-props';
+import { pickInnerProps } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/helpers/pick-inner-props';
+import { pickWrapperProps } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/helpers/pick-wrapper.props';
+import { Outer } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/components/elements/Outer';
+import { Wrapper } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/components/elements/Wrapper';
+import { Inner } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/components/elements/Inner';
+
+const { Provider } = breadcrumbContext;
+
+export function BreadcrumbsItem({
```

Comment:
`BreadcrumbsItem` seems like a better level to apply React's dot notation, i.e. `<Breadcrumbs.Item>`